rbtcwrapperproxy/RBTCWrapperProxy.sol

```diff
@@ -227,4 +251,98 @@ contract RBTCWrapperProxy{
         }        
     }
 
+    function addLiquidityToV1(
+        address _liquidityPoolConverterAddress,
+        IERC20Token[] memory _reserveTokens, 
+        uint256[] memory _reserveAmounts, 
+        uint256 _minReturn
+    )  
+        public
+        payable
```

Comment:
if `wrbtcTokenAddress` is one of `_reserveTokens` the whole transaction would fail for multiple reasons. Probably it is the expected behavior anyway.

Reply:
I have modified the code to require the `wrbtcTokenAddress` to be the first element. I believe front-end guys will do that; And of course, if `_reserveTokens` doesn't have `wrbtcTokenAddress`, the transaction should fail as expected